### ISSUE TITLE
Slideshow dialog - improperly labelled buttons

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1774]
1. x (close) button is only identified as "x"

- [x] replace the x with meaningful text for screen readers. One option is to use an svg in place of the x text, and label it "close"; or do something like this: `<span aria-hidden="true">x</span><span class="sr-only">close</span>`

2. svg icons need context-specific titles that describe what they do, not what they are. Currently the next and previous icons have these titles:
- chevron_left icon (read aloud as "chevron underscore left icon")
- chevron_right icon

- [x] replace the titles for these icons with "previous image" and "next image"

(I wonder if embedding the title in the SVG is a good approach at all, given that the same icon might be used in different contexts? Might be better to use an aria-label in the specific context?)

**Comment by mejackreed** [comment]
With regard to the 2nd item. We could do this using the existing i18n capabilities https://github.com/projectblacklight/blacklight/blob/master/app/models/blacklight/icon.rb#L33

This would have implications everywhere chevron icons are used.

Maybe enhance `Blacklight::Icon` further to take some type of icon context/i18n key so that we can do it more surgically?

**Comment by jkeck** [comment]
FWIW, I ended up filing an issue for this use-case, but for an unrelated a11y issue: projectblacklight/blacklight#2248
